modules/catalog/src/test/java/org/apache/ignite/internal/catalog/CatalogTestUtilsTest.java

```diff
@@ -80,7 +80,7 @@ void testManagerWorksAsExpected() throws Exception {
         assertThat(tablesOfVersion2, hasItem(descriptorWithName("T1")));
         assertThat(tablesOfVersion2, hasItem(descriptorWithName("T2")));
 
-        manager.stop();
+        manager.stopAsync();
```

Comment:
Same as above, won't repeat myself below, please add verification that corresponding future was completed successfully.